x/ccv/provider/client/cli/tx.go

```diff
@@ -44,12 +43,8 @@ func NewAssignConsumerKeyCmd() *cobra.Command {
 				WithTxConfig(clientCtx.TxConfig).WithAccountRetriever(clientCtx.AccountRetriever)
 
 			providerValAddr := clientCtx.GetFromAddress()
-			var consumerPubKey cryptotypes.PubKey
```

Comment:
The entire point of this PR is removing this code. We could unmarshal into a concrete type, but that'd change API